exp/etcdrestore/webhooks/rke2config.go

```diff
@@ -18,16 +18,37 @@ package webhooks
 
 import (
 	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
 
 	bootstrapv1 "github.com/rancher-sandbox/cluster-api-provider-rke2/bootstrap/api/v1beta1"
```

Comment:
We can use this opportunity to update the RKE2 provider dependency after migration to `rancher` organization. What do you think?
```suggestion
	bootstrapv1 "github.com/rancher/cluster-api-provider-rke2/bootstrap/api/v1beta1"
```

Reply:
I needed a new release of CAPRKE2 to make it work, so released https://github.com/rancher/cluster-api-provider-rke2/releases/tag/v0.5.0